### PR TITLE
Fix data race glitch

### PR DESCRIPTION
**Pull request by neelr**

### CODE REVIEW

**Comment by itsmingjie** [comment]
I’ll have a different way to fix it. Closing for now!

File: index.js [hunk 1]
```diff
@@ -201,6 +201,7 @@ var transfer = (
   ts,
   channelid
 ) => {
+  if (user != 'UJYDFQ2QL') return;
```
Comment (cedric-h) [inline index.js:204]: OwO what's this

**Review by cedric-h — CHANGES_REQUESTED** [review]
nvm it's for testing

File: index.js (hunk 1 above)
Comment (neelr) [reply index.js:204]: its to test bro, so only i can make a transaction, and we don't bring banker fully online until I verify that it works

**Comment by neelr** [comment]
> @itsmingjie neel is trying to give himself special perms... this is a douchebag move imo and administrative action should be taken

read above explanation

**Review by cedric-h — APPROVED** [review]

**Pull request merged**